src/AudioToolbox/AudioType.cs

```diff
@@ -456,10 +456,17 @@ public enum AudioChannelLabel : int { // UInt32 AudioChannelLabel
 		HoaAcn14               = (2 << 16) | 14,
 		HoaAcn15               = (2 << 16) | 15,
 		HoaAcn65024            = (2 << 16) | 65024,
+	}
 
-		BeginReserved          = unchecked((int)0xF0000000),
-		EndReserved            = unchecked((int)0xFFFFFFFE),
+#if !COREBUILD
+	public static class AudioChannelLabelExtension
```

Comment:
by conventions (with generated code) you're missing an `s` -> `AudioChannelLabelExtensions`